pkg/types/rekord/v0.0.1/entry.go

```diff
@@ -103,6 +106,23 @@ func (v V001Entry) IndexKeys() []string {
 		}
 	}
 
+	// if this is a certificate, add email addresses to result
+	p, _ := pem.Decode(key)
+	if p != nil {
+		cert, err := x509.ParseCertificate(p.Bytes)
+		if err == nil {
+			var emailAddressOID asn1.ObjectIdentifier = []int{1, 2, 840, 113549, 1, 9, 1}
+			for _, email := range cert.EmailAddresses {
```

Comment:
I think you can just pick one here, probably the cert.Subject.Names. Cosign fills them all in right now and it's a bit silly. Either way they'll always be the same.